packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -589,6 +590,16 @@ export class BasePicker<T, P extends IBasePickerProps<T>>
     }
   };
 
+  /**
+   * Resets focus to last element in wrapper div if clicking back into Picker that has hit item limit
+   */
+  protected onWrapperClick = (ev: React.MouseEvent<HTMLInputElement>): void => {
+    // Reset focus to last item if the input is removed
```

Comment:
_nit:_ This comment can probably go away since we now have a comment for the function above

```suggestion
```